services/application.js

```diff
@@ -78,6 +82,7 @@ class Application {
     }
 
     async submitApplication(params, context) {
+        await this.userService.getMyUser(params, context);
```

Comment:
I think we can remove this getMyUser call. We are expecting that getMyUser will be called at login so it should already be in the session. If it is not then it will fail the verification in the lines below

Reply:
updated